README.md

```diff
@@ -7,17 +7,21 @@ Getting started
 ===============
 
 1. Install or update `node.js` so that your `npm` is at least at version `2.0.0`
-2. Clone the repo: `git clone https://github.com/vector-im/vector-web.git` 
-3. Switch to the SDK directory: `cd vector-web`
-4. Install the prerequisites: `npm install`
-5. Start the development builder and a testing server: `npm start`
-6. Wait a few seconds for the initial build to finish.
-7. Open http://127.0.0.1:8080/ in your browser to see your newly built Vector.
+1. Clone the repo: `git clone https://github.com/vector-im/vector-web.git` 
+1. Switch to the vector directory: `cd vector-web`
+1. Install the prerequisites: `npm install`
+1. Start the development builder and a testing server: `npm start`
+1. Wait a few seconds for the initial build to finish (the command won't
+   terminate: it's running a web server for you).
+1. Open http://127.0.0.1:8080/ in your browser to see your newly built Vector.
 
 With `npm start`, any changes you make to the source files will cause a rebuild so
 your changes will show up when you refresh. This development server also disables
 caching, so do NOT use it in production.
 
+Deployment
+==========
+
```

Comment:
This is now missing information: "run your own server" is vague (in fact we imply `npm run build` will do it for you). You need "mount the `vector` directory on your webserver to actually serve up the app, which is entirely static content."